recipes/ipyleaflet-gl-vector-layer-plugin/meta.yaml

```diff
@@ -16,18 +16,16 @@ build:
 
 requirements:
   host:
-    - python >=3.9
+    - python >=3.6
     - pip
-    - jupyter-packaging
-    - jupyterlab >=3
+    - setuptools >=40.8.0
+    - jupyter-packaging ~=0.7.9
+    - jupyterlab >=3.0.0
     - yarn
-
   run:
-    - python >=3.9
-    - traittypes
-    - traitlets
```

Comment:
traittypes and traitlets are not listed as dependencies upstream. Did you add them b/c they are optional or something?

Reply:
@srepmub if you are OK with mu changes we can merge this one.

Reply:
thanks! the traitlets/traittypes are imported from ipyleaflet_gl_vector_layer/plugin.py.. which should break the import test right, unless they are pulled in by something else I guess. the rest looks fine to me ^^

Reply:
I added these back in, also numpy, as they are imported from the python code. will see if I can get those added upstream for the next release.

Reply:
added to upstream main.